.github/workflows/release.yml

```diff
@@ -184,27 +184,33 @@ jobs:
       matrix:
         python-version: [3.6, 3.7, 3.8]
     steps:
+      - name: Configure Pagefile
+        uses: al-cheb/configure-pagefile-action@v1.2
+        with:
+          minimum-size: 4GB
+      - name: Pagefile size
+        run: |
+          (Get-CimInstance Win32_PageFileUsage).AllocatedBaseSize
+        shell: pwsh
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
       - name: Build Windows wheels
         run: |
-          export LCE_RELEASE_VERSION=${{ github.event.inputs.version }}
```

Comment:
I think we still need this to set the correct version.

Reply:
Ah oops, didn't mean to remove that, thanks :)